tendrl_admin.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Could you change name of this playbook to `qe_tendrl_admin.yml`? So that it would be obvious that this is a qe playbook, not necessary for ci setup or not following official setup documentation.